MantidQt/CustomInterfaces/src/Reflectometry/ReflMainViewPresenter.cpp

```diff
@@ -105,6 +104,8 @@ void ReflMainViewPresenter::notify(IReflPresenter::Flag flag) {
 /** Pushes the list of commands (actions) */
 void ReflMainViewPresenter::pushCommands() {
 
+  m_view->clearCommands();
```

Comment:
Good. In future. Please add specific tests for this kind of thing. It's important to keep the tests in step.